services/frontend-v3/src/components/basicInfo/BasicInfoView.jsx

```diff
@@ -170,8 +181,24 @@ const BasicInfoView = ({
   const getLocationInfo = () => {
     const branch = {
       icon: <BranchesOutlined />,
-      title: <FormattedMessage id="profile.branch" />,
-      description: data.branch ? data.branch : "-",
+      title: <FormattedMessage id="profile.org.tree" />,
+      description: data.branch ? (
+        <Dropdown
+          overlay={
+            <div className="org-dropdown">
+              <OrgTree data={data} />
+            </div>
```

Comment:
the div should instead be a <Menu> since it will properly handle the clicking actions and you'll probably won't need that class anymore

Reply:
Changed to use Menu